modules/ROOT/partials/release-notes/6.0-release-notes-options.adoc

```diff
@@ -41,9 +41,9 @@ NOTE: The `link_default_protocol` value is only applied to an edited or inserted
 [[tinymce-active-editor-options-get]]
```

Comment:
I did wonder about moving this to its own section, since it's about the API changing, rather than options themselves changing?

Reply:
It's more about the option representation so I think this is the correct section.